src/dispatch/experimental/lambda_handler.py

```diff
@@ -0,0 +1,126 @@
+"""Integration of Dispatch programmable endpoints for FastAPI.
+
+Example:
+
+    from dispatch.experimental.lambda_handler import Dispatch
+
+    dispatch = Dispatch(api_key="test-key")
+
+    @dispatch.function
+    def my_function():
+        return "Hello World!"
+
+    @dispatch.entrypoint
+    def entrypoint():
+        my_function()
+
+    def handler(event, context):
+        dispatch.handle(event, context)
+    """
+
+import base64
+import json
+import logging
+from typing import Optional
+
+from dispatch.function import Registry
+from dispatch.proto import Input
+from dispatch.sdk.v1 import function_pb2 as function_pb
+from dispatch.status import Status
+
+logger = logging.getLogger(__name__)
+
+
+class Dispatch(Registry):
+    def __init__(
+        self,
+        api_key: Optional[str] = None,
+        api_url: Optional[str] = None,
+    ):
+        """Initializes a Dispatch Lambda handler.
+
+        Args:
+            api_key: Dispatch API key to use for authentication. Uses the value of
+                the DISPATCH_API_KEY environment variable by default.
+
+            api_url: The URL of the Dispatch API to use. Uses the value of the
+                DISPATCH_API_URL environment variable if set, otherwise
+                defaults to the public Dispatch API (DEFAULT_API_URL).
+
+        """
+
+        super().__init__(endpoint="not configured", api_key=api_key, api_url=api_url)
+
+    def handle(self, event, context):
```

Comment:
Is it possible to add type annotations to these arguments?

Reply:
Maybe? Not sure what will be the types here. Like `Dict[str, Any]` for event maybe?

Reply:
I think this should work:

```
event: str,
context: awslambdaric.lambda_context.LambdaContext
```